patch-package
--- a/node_modules/subscriptions-transport-ws/dist/client.js
+++ b/node_modules/subscriptions-transport-ws/dist/client.js
@@ -47,8 +47,8 @@ var _global = typeof global !== 'undefined' ? global : (typeof window !== 'undef
 var NativeWebSocket = _global.WebSocket || _global.MozWebSocket;
 var Backoff = require("backo2");
 var eventemitter3_1 = require("eventemitter3");
-var is_string_1 = require("./utils/is-string");
-var is_object_1 = require("./utils/is-object");
+var is_string_1 = require("./utils/is-string").default;
+var is_object_1 = require("./utils/is-object").default;
 var printer_1 = require("graphql/language/printer");
 var getOperationAST_1 = require("graphql/utilities/getOperationAST");
 var symbol_observable_1 = require("symbol-observable");
